website/content/docs/sync/awssm.mdx

```diff
@@ -14,37 +14,57 @@ ability to keep control of all your secrets localized while leveraging the benef
 
 ## Setup
 
-The use of this sync destination requires prerequesite setup before beginning to configure it. Either an IAM
-User or an IAM role must be set with the necessary policies in order for Vault to manage secrets in the
-external destination. It is this IAM User or IAM Role in AWS for which access credentials will be generated,
-then provided to Vault during configuration.
-
-The following is an example policy outlining the required permissions to use secrets syncing:
-
-```
-{
-	"Version": "2012-10-17",
-	"Statement": [
-    	{
-    		"Effect": "Allow",
-    		"Action": [
-    			"secretsmanager:Create*",
-    			"secretsmanager:Update*",
-    			"secretsmanager:Delete*",
-    			"secretsmanager:TagResource"
-    		],
-    		"Resource": "arn:aws:secretsmanager:*:*:secret:vault*"
-    	}
-	]
-}
-```
+1.	The use of this sync destination requires prerequesite setup before beginning to configure it. Either an IAM
+	User or an IAM role must be set with the necessary policies in order for Vault to manage secrets in the
+	external destination. It is this IAM user or IAM role in AWS for which access credentials will be generated,
+	then provided to Vault during configuration.
+
+	The following is an example policy outlining the required permissions to use secrets syncing:
+
+	```json
+	{
+		"Version": "2012-10-17",
+		"Statement": [
+			{
+				"Effect": "Allow",
+				"Action": [
+					"secretsmanager:Create*",
+					"secretsmanager:Update*",
+					"secretsmanager:Delete*",
+					"secretsmanager:TagResource"
+				],
+				"Resource": "arn:aws:secretsmanager:*:*:secret:vault*"
+			}
+		]
+	}
+	```
 
-## Usage
+1.	Configure a sync destination with the IAM user credentials created in the previous step:
```

Comment:
Looking at Yoko's PR it seems like the doc team prefers `.` to `:` to end numbered items sentences.
https://github.com/hashicorp/vault/pull/23289/files#diff-fd36a722754c7007a48bb82b51e744558b8815650c1a167f7295e093e189a59bR80